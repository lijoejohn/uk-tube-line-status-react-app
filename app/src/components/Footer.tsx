import { DOM_ELEMENT_IDS, LABELS } from '../constans';

function Footer(): JSX.Element {
  return (
    <footer className="govuk-footer " role="contentinfo">
      <div className="govuk-!-margin-left-9 govuk-!-margin-right-9">
        <div className="govuk-footer__meta">
          <div className="govuk-footer__meta-item govuk-footer__meta-item--grow">
            <span
              data-testid={DOM_ELEMENT_IDS.footer_text}
              id={DOM_ELEMENT_IDS.footer_text}
              className="govuk-footer__licence-description">
              {LABELS.FOOTER_TEXT}
            </span>
          </div>
        </div>
      </div>
    </footer>
  );
}

export default Footer;
