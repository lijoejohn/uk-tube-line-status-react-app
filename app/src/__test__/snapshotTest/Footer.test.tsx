import * as renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('Footer Component', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
