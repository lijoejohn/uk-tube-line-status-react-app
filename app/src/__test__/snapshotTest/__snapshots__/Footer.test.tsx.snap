// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should match the snapshot 1`] = `
<footer
  className="govuk-footer "
  role="contentinfo"
>
  <div
    className="govuk-!-margin-left-9 govuk-!-margin-right-9"
  >
    <div
      className="govuk-footer__meta"
    >
      <div
        className="govuk-footer__meta-item govuk-footer__meta-item--grow"
      >
        <span
          className="govuk-footer__licence-description"
          data-testid="footer_text"
          id="footer_text"
        >
          All content is available under the Sample Licence
        </span>
      </div>
    </div>
  </div>
</footer>
`;
